circle.yml

```diff
@@ -6,7 +6,9 @@ machine:
 
 dependencies:
   override:
+    - brew install swiftlint
     - bundle install
+    - rake danger
```

Comment:
Pretty sure Danger on Circle expects that it might be not set yet, and does the API call, so if she's later in the process, there's a higher chance of that hooking up